app.js

```diff
@@ -0,0 +1,57 @@
+//require all dependencies
```

Comment:
Any reason for the comments in this file?

Reply:
Yes: for me to remember what's happening there!!

Reply:
haha ok! But as you might remember from Clean Code, comments in your are usually not encouraged... Maybe you could find a different way to remember what is happening?